Source/WebCore/Headers.cmake

```diff
@@ -1137,6 +1137,7 @@ set(WebCore_PRIVATE_FRAMEWORK_HEADERS
     dom/Position.h
     dom/ProcessingInstruction.h
     dom/ProgressEvent.h
+    dom/PseudoElement.h
```

Comment:
Why is this necessary?

Reply:
Moved build fixes to https://github.com/WebKit/WebKit/pull/39640.